README.md

```diff
@@ -28,6 +26,12 @@ This exporter package targets [.NET Standard 2.0](https://docs.microsoft.com/en-
 To set up a Dynatrace metrics exporter, add the following code to your project:
 
 ```csharp
+using System.Diagnostics;
```

Comment:
Is it common to have the imports in the snippets? I don't remember that I have seen this before

Reply:
When I was trying the snippets in a new, empty console app, some of the imports aren't offer by VS intellisense. These are often extension methods.. and if you don't know it's hard to find them. So I thought of adding them to the snippet so users don't have to look around :)